internal/credentials/spiffe_test.go

```diff
@@ -231,3 +233,110 @@ func (s) TestSPIFFEIDFromCert(t *testing.T) {
 		})
 	}
 }
+
+var SPIFFETrustBundle map[string]*spiffebundle.Bundle
+
+func TestKnownSPIFFEBundle(t *testing.T) {
+	bundles, err := LoadSPIFFEBundleMap(testdata.Path("spiffe/spiffebundle.json"))
+	if err != nil {
+		t.Fatalf("Error during parsing: %v", err)
```

Comment:
Please make the error strings more actionable by identifying the function and its input. See: go/go-style/decisions#identify-the-function and go/go-style/decisions#identify-the-input